### DISCUSSION TITLE
[Feature] Option to auto loop video

### DISCUSSION THREAD

**Discussion by lordkitsuna** [immich-app/immich#4832] [Feature Request]
### The feature

many "gif" these days are just video files that auto loop. but immich can't do this so cute little 1-4 second loop animations are very difficult to view. an option, possibly just per file as metadata or just global for any video under N seconds to auto loop would be great.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #3289
